docs/installation.rst

```diff
@@ -13,6 +13,11 @@ Other package managers like venv could be used instead of conda, we have just ch
 **NOTE: DESC requires python>=3.9.**
 **If you have python2 also locally installed, replace all `pip` commands with `pip3` and all `python` commands with `python3` to ensure the correct python version is used.**
 
+**NOTE: If you are on Windows, consider using the Windows Subsystem for Linux (WSL) to install DESC.**
+
+We don't test or support DESC on Windows OS, and there have been some instances that numerical discrepancies on Windows can cause failures or wrong results. For instructions on how to install WSL see `here <https://learn.microsoft.com/en-us/windows/wsl/install>`__. For using WSL in VS Code see `here <https://code.visualstudio.com/docs/remote/wsl>`__.
```

Comment:
```suggestion
We don't test or support DESC on Windows OS, and there have been some instances that numerical discrepancies on Windows can cause failures or wrong results. For these reasons, we recommend using WSL if you have a Windows machine. For instructions on how to install WSL see `here <https://learn.microsoft.com/en-us/windows/wsl/install>`__. For using WSL in VS Code see `here <https://code.visualstudio.com/docs/remote/wsl>`__.
```

Reply:
sorry this is my last wording change

Reply:
I assume the `Note:.....` above says that but anyway. I committed it.

Reply:
It was kind of unclear to me, I wanted to make clear we are distinguishing btwn Windows OS (we dont support at all and it prob wont work) and WSL (which technically is linux so it should work fine)